src/Apps/Sell/Routes/AdditionalRoutes/AdditionalDocumentsRoute.tsx

```diff
@@ -31,32 +42,45 @@ interface AdditionalDocumentsRouteProps {
 
 export const AdditionalDocumentsRoute: React.FC<AdditionalDocumentsRouteProps> = props => {
   const submission = useFragment(FRAGMENT, props.submission)
-  const { actions } = useSellFlowContext()
-  // TODO: Add ref to first input
-  // const focusedInputRef = useFocusInput()
 
-  const onSubmit = async (values: FormValues) => {
-    return actions.updateSubmission(values)
-  }
+  const documents: DropzoneFile[] = ((submission.assets as Asset[]) || []).map(
+    dropzoneFileFromAsset
+  )
 
-  const initialValues: FormValues = {
-    title: submission.title ?? "",
+  const initialValues: DocumentsFormValues = {
+    submissionId: submission.externalId,
+    documents,
   }
 
   return (
-    <Formik<FormValues>
+    <Formik<DocumentsFormValues>
       initialValues={initialValues}
-      onSubmit={onSubmit}
+      onSubmit={() => {}}
       validateOnMount
       validationSchema={Schema}
     >
-      {({ handleChange, values }) => (
-        <SubmissionLayout>
-          <SubmissionStepTitle>Additional Documents</SubmissionStepTitle>
+      {({ values }) => {
+        const isAnyDocumentLoading = values.documents.some(
+          (document: DropzoneFile) => document.loading
+        )
+
+        return (
+          <SubmissionLayout loading={isAnyDocumentLoading}>
```

Comment:
Could we use `setLoading` from the context here directly instead of passing it as a prop to `SubmissionLayout`? Same for`PhotosRoute.tsx`.